Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/TrackFindingAlgorithm.hpp

```diff
@@ -97,6 +97,8 @@ class TrackFindingAlgorithm final : public IAlgorithm {
     std::optional<Acts::TrackSelector::Config> trackSelectorCfg = std::nullopt;
     /// Run backward finding
     bool backward = false;
+    /// Maximum number of propagation steps
+    unsigned int maxSteps = 10000000;
```

Comment:
Is 100k steps not by definition a bug? Should we default to 100k even when we make it confiugrable?